Player.py

```diff
@@ -101,3 +112,20 @@ def move(self):
             elif self.state >= 40:
                 self.state = 0
 
+        if keys[pygame.K_SPACE]:
+            if self.bombNumber > 0 and self.bombQueue == 1:
+                self.bombNumber -= 1
+                self.newBomb = Bomb(self.x, self.y, self.bombPower)
+                self.bombQueue = 0
+        else:
+            self.bombQueue = 1
```

Comment:
`bombQueue` means that bomb is in queue or isn't? And even if, why won't you use booleans `True` and `False`? And I don't think that `bombQueue` is a good name. What about `bombPlacedThisRound`?

You could extract these checks to seperate functions like `self.canPlaceBomb()` and in this function you could check for amount of bombs and if the bomb was placed this round.